contents/java/JUnit5.md

```diff
@@ -0,0 +1,137 @@
+<frontmatter>
+  title: Introduction to JUnit 5
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Testing in Java with JUnit 5
+
+Author(s): [Lin Si Jie](https://github.com/sijie123)
+
+## What is JUnit 5?
+JUnit is a open-sourced automated testing framework for Java. Using its vast API library, you can easily write tests to ensure correctness in your software engineering project. After setting up JUnit, running tests within popular IDEs like IntelliJ is as easy as clicking a button. JUnit 5 is the latest iteration of the JUnit project, supporting multiple new features in Java (such as Lambdas) and adopting a new Extensions-based model to make writing tests easier, cleaner, and more modular.
+
+## Why do automated testing?
+There are many reasons why automated testing is important for any software engineering project. Automated testing
+* is the first line of defence against bugs
+* protects against <popover effect="fade" content="changes that unintentionally break existing functionality" placement="top">regression</popover>
+* checks against expected output, preventing errors arising from <popover effect="fade" content="looking manually at output" placement="top">LMAO</popover> or <popover effect="fade" content="looking manually for abnormal output" placement="top">LMFAO</popover>. ([Acronym source](https://stackoverflow.com/questions/10858990/why-use-junit-for-testing))
+* ultimately, giving the developer confidence that their code works correctly.
+
+In fact, testing is so powerful and important that an entire agile development process named Test Driven Development is designed with tests as the centerpiece of software development.
+
+## Why use JUnit 5?
+
+### Benefit: JUnit 5 is the most popular testing framework for Java
+According to [Synk](https://snyk.io/blog/jvm-ecosystem-report-2018-tools/), JUnit is the most popular testing framework for software engineers working in Java. Surveying over 10,200 respondents, an impressive 78% of respondents use JUnit as their primary testing framework.
+<pic src="{{baseUrl}}/java/junit5_popularity.png"></pic>
+
+### Benefit: JUnit 5 is tightly integrated with IntelliJ
+JUnit is tightly integrated with IntelliJ, the most popular IDE (Source: [Synk](https://snyk.io/blog/jvm-ecosystem-report-2018-tools/)) that enjoys over 45% market share.
+With IntelliJ, you can:
```

Comment:
As of now, this benefit seems more like a benefit of using `IntelliJ`, rather than `JUnit 5`. Perhaps you can rephrase the argument to make it seem like a benefit of `JUnit 5`?